connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/CustomSparkConnectBlockingStub.scala

```diff
@@ -27,11 +27,17 @@ private[connect] class CustomSparkConnectBlockingStub(
     retryPolicy: GrpcRetryHandler.RetryPolicy) {
 
   private val stub = SparkConnectServiceGrpc.newBlockingStub(channel)
+
   private val retryHandler = new GrpcRetryHandler(retryPolicy)
 
+  // Constructing GrpcExceptionConverter with a GRPC stub for fetching error details from server.
```

Comment:
I'd remove the `Constructing` bit, because that this is pretty clear if you do `new`,,,,